.github/workflows/ci.yaml

```diff
@@ -92,3 +92,16 @@ jobs:
           file: coverage.xml
           fail_ci_if_error: False
           verbose: True
+          flags: fast-tests
+
+      - name: codecov-schedule
+        if: ${{ github.repository == 'OpenFreeEnergy/openfe'  # we only want to upload a slow report if
+                && github.event_name != 'schedule'            # 1) it isn't a schedule run
```

Comment:
there's a syntax error on this line apparently

Reply:
can't weve comments into the expression it seems, fixed!